Source/ACE/Network/Motion/ServerControlMotion.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using ACE.Network.Enum;
+using ACE.Entity;
+
+namespace ACE.Network.Motion
+{
+    using System;
+
+    public class ServerControlMotion : MotionState
+    {
+        public bool HasTarget { get; set; } = false;
+        public bool Jumping { get; set; } = false;
+        
+        public MotionStance Stance { get; } // command
+
+        public Position Position { get; }
+
+        public ServerControlMotion(MotionStance stance, WorldObject moveToTarget)
+        {
+            Stance = stance;
+            Position = moveToTarget.PhysicsData.Position;
+        }
+
+        /// <summary>
+        /// Used with move to object.
+        /// </summary>
+        /// <param name="moveToTarget">This is the world object we are moving toward</param>
+        /// <param name="distanceToObject">How far away from the object do you want to stop?</param>
+        /// <returns></returns>
+        public byte[] GetPayload(WorldObject moveToTarget, float distanceToObject)
+       {
+            // 4320783 = EE0F - sluggish movement at the end and afterwards 
+            // 4320847 = EE4F - sluggish movement at the end and afterwards
+            // 4321264 = EFF0 - sluggish movement at the end and afterwards
+            // 4321136 = EF70 - sluggish movement at the end and afterwards
+            // 4319823 = EA4F
+            // EE9F -? fail distance 100
+            const uint Flag = 0x0041EE0F;  
```

Comment:
Will do.    I am hoping I can understand how that bit field is constructed so it may be built correctly and not be a const.   If there is no magic solution to that, I will name and move the constants to the correct location.   Thanks for the pointer.